core/common/src/tile/TileMetadata.ts

```diff
@@ -60,6 +60,7 @@ export interface TileOptions {
   readonly ignoreAreaPatterns: boolean;
   readonly enableExternalTextures: boolean;
   readonly useProjectExtents: boolean;
+  readonly optimizeBRepProcessing: boolean;

```

Comment:
@paulius-valiunas @wgoehrig note this addition to TileOptions and implications for tile agents.